ui/src/app/job-details/tasks/tasks.component.css

```diff
@@ -74,3 +76,38 @@ clr-icon {
 ::ng-deep clr-icon.log-error.disabled .clr-i-alert.clr-i-outline--alerted {
   fill: currentColor;
 }
+
+a.title-link, p.title-nolink {
+  display: inline-flex;
+}
+
+button.shard-icon {
+  display: inline-flex;
+  margin-left: 1.5rem;
+}
+
+.shards-container {
+  display: inline-flex;
+  flex-direction: row;
+}
+
+.shards-header {
+  font-size: 0.8rem;
+  line-height: 1.5rem;
+  margin: 0 0.5rem 0 0.5rem;
+  font-weight: bold;
+}
+
+.shards-statuses {
+  margin: 0.5rem;
+}
+
+.shard-count {
```

Comment:
ToL (naive question because I am 100% *not* a UI person):

Does it make sense to be this granular with styles? Is there not a general purpose "info box number" style, or similar?